package.json

```
@@ -13,9 +13,9 @@
   "homepage": "https://github.com/Brightspace/user-tile",
   "name": "user-tile",
   "scripts": {
-    "lint": "npm run lint:wc && npm run lint:js",
+    "lint": "npm run lint:js",
```

```suggestion
    "lint": "npm run lint:lit-analyzer && npm run lint:js",
```

I think we still want to run `lit-analyzer` in our general lint step/CI in order to [statically catch some LitElement-specific issues](https://www.npmjs.com/package/lit-analyzer#%E2%9E%A4-rules) - just a naming fixup.